packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -0,0 +1,83 @@
+// This script deploys SlashingLibrary, VotingV2 and VotingUpgrader.
+// This script can be run against a mainnet fork by spinning a node in a separate terminal with:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// and then running this script with:
+// yarn hardhat run ./src/upgrade-tests/voting2/0_Deploy.ts --network localhost
+
+const hre = require("hardhat");
+
+import { VotingUpgrader__factory } from "@uma/contracts-frontend/dist/typechain/core/ethers";
```

Comment:
two questions:
1) why does this come from contracts-front end not node?
2) do you need to deep path like this into the package? would it not work to just go `import { VotingUpgrader__factory } from "@uma/contracts-frontend`?

Reply:
+1. We should avoid deep imports. If we end up _needing_ deep imports, that likely means something is wrong with the downstream project and it's not exporting everything correctly.

Reply:
Fixed in https://github.com/UMAprotocol/protocol/pull/4132